### PR TITLE
should get return err and check it

### PR DESCRIPTION
**Pull request by hanxiaoshuai**
**What this PR does / why we need it**:
should get return err and check it
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/auth/certificates.go [hunk 1]
```diff
@@ -91,7 +91,7 @@ var _ = SIGDescribe("Certificates API", func() {
 
 		framework.Logf("waiting for CSR to be signed")
 		framework.ExpectNoError(wait.Poll(5*time.Second, time.Minute, func() (bool, error) {
-			csr, _ = csrs.Get(csrName, metav1.GetOptions{})
+			csr, err = csrs.Get(csrName, metav1.GetOptions{})
 			if err != nil {
 				return false, err
```
Comment (mikedanese) [inline test/e2e/auth/certificates.go]: log the error and return false, nil

Comment (hanxiaoshuai) [reply test/e2e/auth/certificates.go]: Done

**Pull request merged**